LeanSAT/Model/Subst.lean

```diff
@@ -88,6 +209,26 @@ theorem PropFun.satisfies_subst [DecidableEq ν₁]
   simp at this
   exact this.symm
 
+-- CC: Unsure how to prove for fancy `subst`.
+--@[simp] theorem subst_distrib : subst ⟦v⟧ f = f v := by
```

Comment:
`by  ext; simp [PropAssignment.subst]` :D

Reply:
We love short proofs that we miss initially.

Reply:
Proof added.